docs/guides/src/partitioning.md

```diff
@@ -0,0 +1 @@
+# Partitioning
```

Comment:
Partition Table
```suggestion
# Partition Table
```

Reply:
Maybe we should call it `Table Partitioning`

Reply:
I guess `Table Partitioning` seems better.